BUILDING.md

```diff
@@ -1,34 +1,106 @@
+# Using GitHub actions
+
+In the GitHub medley repository (Interlisp/medley) go to the Actions tab.
+
+It will list the available GitHub actions.
+Select: **Build/Push Release & Docker**. 
+
+In the middle of the screen there is a box labeled "Workflow Runs".
+There should be a row in it that states 'This workflow has a workflow_dispatch event trigger' with a drop down menu (it really looks more like a button) on the right side labeled 'Run workflow'.  Select that and you'll get a form allowing you to select the branch (I've only used Master) and enter the release name.  Enter a name or leave it empty and press the green 'Run workflow' button. The workflow should queue up and run.  
+
+Build/Push Release & Docker first builds Maiko and Medley, pushes a Medley release to the Interlisp/medley repo Releases, then makes a Docker image.
+
+The files in .github/workflows/ contain the details.
+<!-- 
+The workflow pulls the latest Maiko image from Docker Hub and the Release Assets from the latest Medley release, generally defined as medley-YYMMDD.  The Medley Docker image adds in Tight VNC Server and retrieves the two tarballs associated with a release, one containing the sysouts and the other the other needed files source, fonts, etc.  The contents are uncompressed and loaded into the Medley directory structure.
+-->
+
+# Building Medley for yourself
+
+The actual building itself is done with various shell scripts, found in the `scripts` directory in the medley repository. Most of the scripts have a minimal sanity check that they are being run from the medley repository.
+
+Historically, building the medley image (called a "sysout") was called "doing a loadup".  Back in the day, a loadup took the better part of a day, and no one would do the whole thing -- there was no automation.
+
+## Prerequisites
+
+In these instructions, there is an assumption that the loadup scripts can find other repositories. All of the loadup scripts have to find `lde` and `ldex` or `ldisdl` and also `ldeinit`.
+
+Loadups use the run-medley script, which looks for Maiko (actually the lde & ldeinit executables) as follows:
+
+1. lde (ldeinit) on PATH
+2. In the `<osversion>.<machinetype>` subdirectory of the directory specified by $MAIKODIR
+3. In the `<osversion>.<machinetype>` subdirectory of the directory specified by $MEDLEYDIR/../maiko/
+4. In the `<osversion>.<machinetype>` subdirectory of the directory specified by $MEDLEYDIR/maiko/
+
+where $MEDLEYDIR is the directory from which you called the loadup script.
+
+## Make everything
+
+The shell command:
 ```
-./scripts/loadup-and-release.sh
+time ./scripts/loadup-all.sh -apps && time ./scripts/loadup-db.sh
 ```
-to go on to make the tgz files and release them
+does everything; on a fast machine it takes 4-6 minutes, most of which is spent in the `loadup-db.sh` step. Without the `loadup-db.sh` step, it runs in 22 seconds on a fast system.
 
-# Using github actions
+These are explained in reverse order:
```

Comment:
A few lines above the document advises to run `loadup-all.sh` and `loadup-db.sh` to make everything. So where do the scripts described below fit in? Should the user run them explicitly? Under what circumstances should the scripts be run?